tests/pass/no_std.stderr

```diff
@@ -0,0 +1,91 @@
+thread 'rustc' panicked at 'failed to find required Rust item: ["std", "sys", "windows", "thread_local_key", "p_thread_callback"]', src/helpers.rs:LL:CC
```

Comment:
This file doesn't look like it should be here?

Reply:
Yeah, if the output is empty in bless mode it should probably delete a potentially existing file.

Reply:
removed and squashed commits